runner/src/load_generator/gen.rs

```diff
@@ -0,0 +1,233 @@
+
+    Ok(CommandResult::Success)
+}
+
+/**
+ * Create model instances continuously
+ *
+ * @param stable_load_user The user to create the model instances
+ * @param model The model schema to create model instances from
+ * @param duration_in_hours The duration to run the simulation in hours
+ * @return The result of the simulation
+ */
+pub async fn create_model_instances_continuously(
+    stable_load_user: StableLoadUser,
+    model: StreamId,
+    duration_in_hours: u64,
+) -> Result<()> {
+    let start_time = Instant::now();
+
+    let duration = Duration::from_secs(duration_in_hours * 60 * 60);
+    let mut count = 0;
+    let mut error_map: HashMap<String, u64> = HashMap::new();
+    // TODO : Make the rps configurable
+    // TODO : Make the channel size configurable
+    // TODO : Make the number of tasks configurable : tasks are currently 100 -
+    // increasing tasks can help increase throughput
+    let (tx, mut rx) = tokio::sync::mpsc::channel(10000);
+    let mut tasks = tokio::task::JoinSet::new();
+    for i in 0..100 {
+        let user_clone = stable_load_user.clone();
+        let model = model.clone();
+        let tx = tx.clone();
+        tasks.spawn(async move {
+            loop {
+                if start_time.elapsed() > duration {
+                    println!("loop {i} Duration expired");
+                    break;
+                }
+                match tokio::time::timeout(
+                    Duration::from_secs(5),
+                    user_clone.ceramic_utils.create_random_mid(&model),
+                )
+                .await
+                {
+                    Ok(Ok(mid)) => match tx.send(Ok(mid.to_string())).await {
+                        Ok(_) => {}
+                        Err(e) => {
+                            println!("Failed to send MID: {}", e);
+                        }
+                    },
+                    Ok(Err(e)) => match tx.send(Err(e.to_string())).await {
+                        Ok(_) => {}
+                        Err(e) => {
+                            println!("Failed to send error: {}", e);
+                        }
+                    },
+                    Err(e) => match tx.send(Err(e.to_string())).await {
+                        Ok(_) => {}
+                        Err(e) => {
+                            println!("Failed to send error: {}", e);
+                        }
+                    },
+                }
+            }
+        });
+    }
+    drop(tx);
+    loop {
+        let mut mid_vec: Vec<Result<String, String>> = Vec::new();
+        if rx.recv_many(&mut mid_vec, 10).await > 0 {
+            for mid in mid_vec {
+                match mid {
+                    Ok(_) => {
+                        count += 1;
+                    }
+                    Err(err) => {
+                        *error_map.entry(err).or_insert(0) += 1;
+                    }
+                }
+            }
+        }
+        if start_time.elapsed() > duration {
```

Comment:
it might be nice to add at least a few seconds to this so you give the tasks a chance to exit gracefully as they're waiting for the exact same time. I'd suggest at least 5 since that's the timeout above